frontend/app/views/fragments/event/item.scala.html

```diff
@@ -34,7 +34,7 @@
     </div>
 }
 
-<a href="@routes.Event.details(event.underlying.ebEvent.slug).url" class="@RenderClasses(Map(
+<a href="@event.detailsUrl" class="@RenderClasses(Map(
```

Comment:
This should have been using the property from the class anyway, rather than distributing the logic to do a slug-based lookup.